app/src/main/java/org/mozilla/fenix/FeatureFlags.kt

```diff
@@ -110,4 +110,9 @@ object FeatureFlags {
             listOf("en-US", "es-US").contains(langTag) && Config.channel.isNightlyOrDebug
         }
     }
+
+    /**
+     * Enables the Task Continuity enhancements.
+     */
+    val taskContinuityFeature = Config.channel.isDebug
```

Comment:
Does sticky headers really need to be behind a feature flag?

Reply:
It appears so.

Reply:
Yeah, these changes are a part of the larger "Task Continuity" enchancements